src/app/model/shared/response.ts

```diff
@@ -0,0 +1,23 @@
+import { Cacke } from "../cacke/cacke";
+import { Request } from "express";
+import { Sell } from "../sell/sell";
+
```

Comment:
On this challenge we are following a single responsibility file, then you should separate all the elements by file.